### PR TITLE
Unpack support

### PR DESCRIPTION
**Pull request by ScottMcMichael**
- Added initial Sentinel 1 support (need to add preprocessing)
- Added batch data unpack tool for WV and S1
- Possibly fixed epoch processing bug

### CODE REVIEW

File: delta/extensions/sources/sentinel1.py [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+
+"""
+Functions to support Sentinel1 satellites.
+"""
+
+import os
+import sys
+import portalocker
+
+import tensorflow as tf
```
Comment (bcoltin) [inline delta/extensions/sources/sentinel1.py:26]: Your pytest error is from this (and the same in worldview file), you can just use normal print here. Also merge in my latest changes, it fixes another build error we were having from an h5py update.

**Comment by bcoltin** [comment]
Is this ready to merge? Also how intensive is this preprocessing? Maybe we should do it before it goes to DELTA if it takes significant time and involves writing files?

**Comment by ScottMcMichael** [comment]
There is an unpacking script which I run before training which is the same as we are doing for Worldview.  At this point merging can probably wait until we see how the results are.

**Comment by bcoltin** [comment]
Sounds great, thanks!

**Comment by ScottMcMichael** [comment]
Ready to merge now.

**Pull request merged**